Dockerfile

```diff
@@ -19,13 +17,8 @@ RUN /opt/conda/envs/rapids/bin/pip install \
     sqlalchemy \
     openpyxl \
     tabulate \
-    autopep8
-
-# misc python packages
-RUN conda install -n rapids pywget
-
-# plotly
-RUN conda install -n rapids -c plotly plotly=4.9.0
+    autopep8 \
+    chembl_webresource_client
 
 RUN /opt/conda/envs/rapids/bin/pip install --ignore-installed --upgrade \
```

Comment:
Should we fully remove the code for the cddd model? That would include tensorflow and model download in the Dockerfile, plus the embedding class.

Reply:
At the moment that is the only alternate embedding function/model (apart from Morgan Fingerprint) to talk about. Though not useful, it highlights the ability to plugin new embedding models. Lets remove it once we have a better replacement.